client/components/recurring-payment-icon/index.js

```diff
@@ -0,0 +1,64 @@
+import { __ } from '@wordpress/i18n';
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { Popover as PopoverComponent } from '@wordpress/components';
+import icon from './icon.svg';
+
+const Icon = styled.img`
+	height: 12px;
+	width: 12px;
+	cursor: pointer;
+`;
+
+const StyledPopover = styled( PopoverComponent )`
+	top: -11px !important;
+
+	.components-popover__content {
+		border-radius: 4px;
+		box-shadow: 0px 2px 6px 0px rgba( 0, 0, 0, 0.05 );
+		padding: 5px 10px;
+		background-color: #000000;
+		color: #ffffff;
+	}
+
+	@media ( min-width: 660px ) {
+		.components-popover__content {
+			width: auto;
+		}
+	}
+`;
+
+const IconComponent = ( { children, ...props } ) => (
+	<span { ...props }>
+		<Icon src={ icon } alt="" />
+		{ children }
+	</span>
```

Comment:
This `span` element means that we're not getting automatic alignment from the `div` with `inline-flex` that wraps the elements, and we end up with slightly offset/weird vertical alignment.

More specifically, I think the icon should be horizontally aligned with the text, and the current layout has it slightly above that.

Reply:
Initially fixed it in [97d80d7](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4299/commits/97d80d7a55c7d5f09138a9fcb7bc13a3677b1a6d) but the changes are not relevant anymore after updating the code to use the tooltip component [facce12](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4299/commits/facce12b5604d9f84ce10561b5b120093df918ce).